testsuite/integration-deprecated/src/test/java/org/keycloak/testsuite/broker/AbstractFirstBrokerLoginTest.java

```diff
@@ -272,37 +272,6 @@ public void testFixDuplicationsByReviewProfile() {
         assertFederatedUser("testing-user", "testing-user@redhat.com", "pedroigor");
     }
 
-    /**
-     * Tests that duplication is detected and user wants to link federatedIdentity with existing account. He will confirm link by email
-     */
-    @Test
-    public void testLinkAccountByEmailVerification() throws Exception {
```

Comment:
Removed in favor of org.keycloak.testsuite.broker.AbstractBrokerTest#testLinkAccountWithEmailVerified.